components/public-api-server/pkg/apiv1/oidc.go

```diff
@@ -318,3 +326,28 @@ func toDbOIDCSpec(oauth2Config *v1.OAuth2Config, oidcConfig *v1.OIDCConfig) db.O
 		Scopes:       append([]string{goidc.ScopeOpenID, "profile", "email"}, oauth2Config.GetScopes()...),
 	}
 }
+
+func assertIssuerIsReachable(host string) error {
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+		Proxy:           http.ProxyFromEnvironment,
```

Comment:
Do we need this? The environment this runs is in controlled by us, and we don't have a system level proxy configured.

Reply:
Took this from a k8s reachability probe.

> we don't have a system level proxy configured.

Since https://github.com/gitpod-io/gitpod/pull/12726 we have an option to set it on the installation level.

> The environment this runs is in controlled by us

I'm not sure this is the question. If the Org only provides a proxy to reach internal Git, which was several times the case with self-hosted, it might still be relevant.